server/test/services/applicant/question/MultiSelectQuestionTest.java

```diff
@@ -25,6 +25,10 @@
 
 public class MultiSelectQuestionTest {
 
+  // TODO(#4862): Once the admin name is assignable and different then the
```

Comment:
Here and elsewhere:
There's nothing to test right now because `QuestionOption` just assigns the default locale's text to the admin name. Once the admin name is assignable, we'll be able to test that we get the admin name back and not the default locale's text.